lang/fr-fr.js

```diff
@@ -35,6 +35,10 @@ export default {
 	"components.input-date.errorOutsideRange": "La date doit être comprise entre {minDate} et {maxDate}",
 	"components.input-date.openInstructions": "Utiliser le format de date {format}. Utilisez la flèche vers le bas ou appuyez sur Entrée pour accéder au mini-calendrier.",
 	"components.input-date.setToToday": "Défini sur Aujourd'hui",
+	"components.input-number.hintInteger": "Ce champ accepte uniquement les valeurs entières (pas de décimales)",
```

Comment:
"champ" is a literal translation of "field" (like a farmer's field). We're looking for field as in "enter your name in this field". @jeff-shepstone would we be able to get this re-translated?

I think this happened to some of the other languages too ("campo" in Portuguese/Spanish) but I'm no expert in those languages so can't really tell.

Reply:
Hey @dlockhart this is also the same translation used in this context :)
![image](https://user-images.githubusercontent.com/15015462/116448494-30bfd000-a827-11eb-9701-b9af51490196.png)

Reply:
Ok excellent! Google translate was confusing me with all the different options:

![Screen Shot 2021-04-28 at 1 55 22 PM](https://user-images.githubusercontent.com/5491151/116450552-6f568a00-a829-11eb-8b83-90750cf3be19.png)